src/Xrefcheck/Verify.hs

```diff
@@ -241,19 +246,51 @@ instance Given ColorMode => Buildable VerifyError where
       #{err}
       |]
 
-    PermanentRedirectError url Nothing ->
+    RedirectChainCycle chain ->
+      [int||
+      Cycle found in the following redirect chain:
+      #{interpolateIndentF 2 $ buildRedirectChain chain ["^-- here"]}
+      |]
+
+    RedirectChainBroken chain ->
       [int||
-      Permanent redirect found:
-      #{url}
+      Missin location header in the following redirect chain:
```

Comment:
I'm not sure if it is necessary because there is a `^-- here` indication pointing to the last link in the chain, but I can modify it if you prefer.

Actually, there is no example with that output in the Golden tests. I will try to find an example to include it.